partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/predicate/OpComparisonTest.kt

```diff
@@ -52,14 +52,17 @@ class OpComparisonTest : PartiQLTyperTestBase() {
 
         ).map { inputs.get("basics", it)!! }
 
+        /**
+         * 8.2 Comparison Predicate — SQL-99 page 287
+         */
         val argsMap = buildMap {
             val successArgs =
                 cartesianProduct(
                     StaticType.NUMERIC.allTypes + listOf(StaticType.NULL),
                     StaticType.NUMERIC.allTypes + listOf(StaticType.NULL)
                 ) + cartesianProduct(
-                    StaticType.TEXT.allTypes + listOf(StaticType.CLOB, StaticType.NULL),
-                    StaticType.TEXT.allTypes + listOf(StaticType.CLOB, StaticType.NULL)
+                    StaticType.TEXT.allTypes + listOf(StaticType.NULL),
+                    StaticType.TEXT.allTypes + listOf(StaticType.NULL)
```

Comment:
Is this because SQL: 99 only support equal / not equal operator for comparison between large object strings?